mods/cnc/maps/gdi08b/gdi08b.lua

```diff
@@ -0,0 +1,208 @@
+Auto8 = { units = { ['e4'] = 2 + SquadsizeBonusMax2[Difficulty], ['e3'] = 4 + SquadsizeBonusMax2[Difficulty] }, waypoints = WaypointGroup1, delay = AutocreateDelay[Difficulty] }
+
+AutoAttackWaves = { Auto2, Auto3, Auto4, Auto5, Auto6, Auto7, Auto8 }
+
+StationaryGuardUnits = { Actor237, Actor238, Actor231, ltnk1, ltnk2, Actor233, ltnk3, arty1, arty2, arty3, bggy, Actor240, Actor242, Actor243, Actor227, Actor228, Actor229 }
+
+StartStationaryGuards = function(StationaryGuards)
+	Utils.Do(StationaryGuards, function(unit)
+		if not unit.IsDead then
+			unit.Patrol( { unit.Location } , true, 20)
+		end
+	end)
+end
+
+SendWaves = function(counter, Waves)
+	if counter <= #Waves then
+		local team = Waves[counter]
+
+		for type, amount in pairs(team.units) do
+			MoveAndHunt(Utils.Take(amount, Nod.GetActorsByType(type)), team.waypoints)
+		end
+
+		Trigger.AfterDelay(DateTime.Seconds(team.delay), function() SendWaves(counter + 1, Waves) end)
+	end
+end
+
+SendHeli = function(heli)
+	units = Reinforcements.ReinforceWithTransport(Nod, "tran", heli.types, heli.entry, { heli.entry[1] })
+	Utils.Do(units[2], function(actor)
+		actor.Hunt()
+		Trigger.OnIdle(actor, actor.Hunt)
+	end)
+	if heli.delay == DateTime.Seconds(0) then
+		return
+	end
+	Trigger.AfterDelay(heli.delay, function() SendHeli(heli) end)
+end
+
+MoveInitialArty = function(arty, waypoints)
+	units = { arty }
+	MoveAndIdle(units, waypoints)
+end
+
+MoveInitialArty2 = function()
+	units = { earlyarty2 }
+	MoveAndIdle(units, ArtyWaypoints2)
+end
+
+TankTerror = function(tank)
+	units = { tank }
+	MoveAndHunt(units, WaypointGroup1)
+end
+
+WorldLoaded = function()
+	GDI = Player.GetPlayer("GDI")
+	Nod = Player.GetPlayer("Nod")
+
+	Camera.Position = DefaultCameraPosition.CenterPosition
+
+	StartStationaryGuards(StationaryGuardUnits)
+
+	StartAI()
+
+	Trigger.OnObjectiveAdded(GDI, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+
+	Trigger.OnObjectiveCompleted(GDI, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+
+	Trigger.OnObjectiveFailed(GDI, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	Trigger.OnPlayerWon(GDI, function()
+		Media.PlaySpeechNotification(Nod, "Win")
+	end)
+
+	Trigger.OnPlayerLost(GDI, function()
+		Media.PlaySpeechNotification(Nod, "Lose")
+	end)
+
+	ProtectMoebius = GDI.AddObjective("Protect Dr. Mobius.")
+	Trigger.OnKilled(DrMoebius, function()
+		GDI.MarkFailedObjective(ProtectMoebius)
+	end)
+	
+	ProtectHospital = GDI.AddObjective("Protect the Hospital.")
+	Trigger.OnKilled(Hospital, function()
+		GDI.MarkFailedObjective(ProtectHospital)
+	end)
+	
+	ProtectCivilians = GDI.AddObjective("Protect the Civilians.")
```

Comment:
This would be more informative if it displayed the number of civilians required per difficulty level.